Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_SurfaceData.azsli

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// This #define magic lets you use the Surface class in this file without making it the final Surface
+// used in your shader. Simply #define Surface to your custom definition before including this file
+//
+#ifndef Surface
+#define Surface     SurfaceData_BasePBR
+#endif
+
+#include <Atom/Features/PBR/MaterialUtils.azsli>
+#include <Atom/Features/PBR/SpecularAA.azsli>
+
+class SurfaceData_BasePBR
+{   
+    precise float3 position;    //!< Position in world-space
+    float3 normal;              //!< Normal in world-space
+    float3 vertexNormal;        //!< Vertex normal in world-space
+    float3 baseColor;           //!< Surface base color
+    float metallic;             //!< Surface metallic property
+    float roughnessLinear;      //!< Perceptually linear roughness value authored by artists. Must be remapped to roughnessA before use
+    float roughnessA;           //!< Actual roughness value ( a.k.a. "alpha roughness") to be used in microfacet calculations
+    float roughnessA2;          //!< Alpha roughness ^ 2 (i.e. roughnessA * roughnessA), used in GGX, cached here for perfromance
+
+    //! Surface lighting data
+    float3 albedo;              //!< Albedo color of the non-metallic material, will be multiplied against the diffuse lighting value
+    float3 specularF0;          //!< Fresnel f0 spectral value of the surface
+
+    //! Calculates roughnessA and roughnessA2 after roughness has been set
+    void CalculateRoughnessA();
+
+    //! Sets albedo, base color, specularF0, and metallic properties using metallic workflow
+    void SetAlbedoAndSpecularF0(float3 newBaseColor, float specularF0Factor, float newMetallic);
+};
+
+void SurfaceData_BasePBR::CalculateRoughnessA()
```

Comment:
Nit: Suggest `SetRoughnessLinear(float roughnessLinearIn)` as opposed to expecting user to set that value prior to invoking this function (maybe a future refactor)

Reply:
Yep, can be a future refactor. I think I considered this at one point but there was some use case against it, maybe to do with multilayer PBR. But can revisit and change if we can't find a reason not to.